### PR TITLE
ZON-5647: py3 zeit.connector

### PR DESCRIPTION
**Pull request by tomster**
WIP

### CODE REVIEW

File: core/src/zeit/connector/mock.txt [hunk 1]
```diff
@@ -158,7 +159,7 @@ We now can get the object back from the connector:
 
 >>> from pprint import pprint
 >>> resource_2 = connector[resource.id]
->>> print(resource_2.data.read())
+>>> print(resource_2.data.read().decode('utf-8'))
```
Comment (tomster) [inline core/src/zeit/connector/mock.txt]: ok, das merk ich mir!

File: core/src/zeit/connector/connector.py [hunk 2]
```diff
@@ -78,12 +78,6 @@ def __repr__(self):
         return '<CannonicalId %s>' % super(CannonicalId, self).__repr__()
 
 
-def stringify(data):
```
Comment (tomster) [inline core/src/zeit/connector/connector.py]: turns out, https://six.readthedocs.io/index.html#six.ensure_text was exactly what i was looking for, at the time... auch hier gilt: merk ich mir!

File: core/src/zeit/connector/connector.txt [hunk 3]
```diff
@@ -467,7 +467,7 @@ inconsistent/wrong state:
 
 >>> conn = connector.get_connection()
 >>> conn._con
-<httplib.HTTPConnection instance at 0x...>
+<...HTTPConnection ... at 0x...>
```
Comment (tomster) [inline core/src/zeit/connector/connector.txt]: leading dots work? how did i miss that all these years?!

**Comment by tomster** [comment]
sieht gut aus, aber FTR ich habe lokal noch ```Results (65.71s):
      43 passed
       5 failed
         - vivi/core/src/zeit/connector/testing.py:268 TestConflictDetectionReal.test_explicit_override
         - vivi/core/src/zeit/connector/testing.py:268 TestConflictDetectionReal.test_no_conflict_with_same_content
         - vivi/core/src/zeit/connector/tests/test_connector.py:177 TestResource.test_properties_should_be_current
         - test_suite:? /vivi/core/src/zeit/connector/connector.txt
         - test_suite:? /vivi/core/src/zeit/connector/uuid.txt
       1 xfailed
      68 deselected

```

File: core/src/zeit/connector/connector.txt (hunk 3 above)
Comment (wosc) [reply core/src/zeit/connector/connector.txt]: Sind ja nicht leading. Das erste Zeichen ist `<` :)

**Comment by tomster** [comment]
ok, the same failures happen for me on python2 and on master, so it's not related to this PR

**Pull request merged**